### PR TITLE
Fix/code style

### PR DESCRIPTION
**Pull request by Mpdreamz**
Serves as a preview to discuss the changes from #46

### CODE REVIEW

File: src/Elastic.Apm.AspNetCore/ApmMiddleware.cs [hunk 1]
```diff
@@ -1,112 +1,115 @@
...
-using Elastic.Apm.Model.Payload;
 using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
 using Elastic.Apm.Helpers;
+using Elastic.Apm.Model.Payload;
+using Microsoft.AspNetCore.Http;
 
-[assembly: InternalsVisibleTo("Elastic.Apm.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010051df3e4d8341d66c6dfbf35b2fda3627d08073156ed98eef81122b94e86ef2e44e7980202d21826e367db9f494c265666ae30869fb4cd1a434d171f6b634aa67fa8ca5b9076d55dc3baa203d3a23b9c1296c9f45d06a45cf89520bef98325958b066d8c626db76dd60d0508af877580accdd0e9f88e46b6421bf09a33de53fe1")]
-[assembly: InternalsVisibleTo("Elastic.Apm.AspNetCore.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010051df3e4d8341d66c6dfbf35b2fda3627d08073156ed98eef81122b94e86ef2e44e7980202d21826e367db9f494c265666ae30869fb4cd1a434d171f6b634aa67fa8ca5b9076d55dc3baa203d3a23b9c1296c9f45d06a45cf89520bef98325958b066d8c626db76dd60d0508af877580accdd0e9f88e46b6421bf09a33de53fe1")]
+[assembly:
```
Comment (Mpdreamz) [inline src/Elastic.Apm.AspNetCore/ApmMiddleware.cs]: This is unfortunate but is due to our `wrap` of `150` I think.

File: sample/SampleAspNetCoreApp/wwwroot/lib/bootstrap/dist/js/npm.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-require('../../js/collapse.js')
-require('../../js/dropdown.js')
-require('../../js/modal.js')
-require('../../js/tooltip.js')
-require('../../js/popover.js')
-require('../../js/scrollspy.js')
-require('../../js/tab.js')
-require('../../js/affix.js')
\ No newline at end of file
+require('../../js/transition.js');
```
Comment (Mpdreamz) [inline sample/SampleAspNetCoreApp/wwwroot/lib/bootstrap/dist/js/npm.js]: Code cleanup also touches `js` files. I reckon we should not check this in and leave these changes as is. cc @elastic/dotnet

File: src/Elastic.Apm/Api/ISpan.cs [hunk 3]
```diff
@@ -5,97 +5,97 @@
...
-    }
-}
\ No newline at end of file
+	public interface ISpan
+	{
+		/// <summary>
+		/// The action of the span.
+		/// Examples: 'query'.
+		/// </summary>
+		string Action { get; set; }
```
Comment (Mpdreamz) [inline src/Elastic.Apm/Api/ISpan.cs]: Code cleanup orders properties/fields alphabetically.

File: src/Elastic.Apm.AspNetCore/ApmMiddleware.cs (hunk 1 above)
Comment (gregkalapos) [reply src/Elastic.Apm.AspNetCore/ApmMiddleware.cs]: I'd say this is acceptable. Not a big deal... Alternative solution: we could put the `PublicKey` int a `consts`.

File: src/Elastic.Apm/Config/AbstractAgentConfig.cs [hunk 4]
```diff
@@ -4,107 +4,100 @@
...
-
-                void AddDefaultWithDebug()
-                {
-                    serverUrls.Add(ConfigConsts.DefaultServerUri);
-                    Logger?.LogDebug($"Using default ServerUrl: {ConfigConsts.DefaultServerUri}");
-                }
-            }
-        }
-
-        protected LogLevel? logLevel;
```
[the remark this answers is not in the document]
Comment (gregkalapos) [reply src/Elastic.Apm/Config/AbstractAgentConfig.cs]: Weeeell ...that's not very nice :(

File: src/Elastic.Apm/Api/ISpan.cs (hunk 3 above)
Comment (gregkalapos) [reply src/Elastic.Apm/Api/ISpan.cs]: That's fine.

File: sample/SampleAspNetCoreApp/wwwroot/lib/bootstrap/dist/js/npm.js (hunk 2 above)
Comment (gregkalapos) [reply sample/SampleAspNetCoreApp/wwwroot/lib/bootstrap/dist/js/npm.js]: +1

**Pull request closed without merging**